ui/package.json

```diff
@@ -152,7 +182,7 @@
     "pkijs": "^2.2.2",
     "pretender": "^3.4.3",
     "prettier": "2.6.2",
-    "prettier-eslint-cli": "^5.0.0",
+    "prettier-eslint-cli": "^7.1.0",
```

Comment:
This was preventing TS from working out of the box because it required an older version.